### PR TITLE
{mpi}[GCC/6.3.0-2.27] MVAPICH2 v2.2

### PR DESCRIPTION
**Pull request by henkela**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by henkela** [comment]
Test report by @henkela
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login23 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/3e7518d9fdbbecf7e7f72a7e0b4e27c2 for a full test report.

File: easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2-GCC-6.3.0-2.27-hydra-opa.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+rdma_type = ''
+
+sanity_check_paths = {
+    'files': ["bin/mpiexec.hydra"],
+     'dirs' :[]
+}
+
+# Configure options
+configopts = [
+	"--sysconfdir=/etc \
```
Comment (boegel) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2-GCC-6.3.0-2.27-hydra-opa.eb]: @henkela This looks quite site-specific...

Also, it shouldn't be a list (with just a single entry), but a string. We usually just list configure options on a single, or spread across multiple lines if it gets too long:

```python
configopts = "--sysconfdir=/etc ..."
configopts += "..."

Comment (henkela) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2-GCC-6.3.0-2.27-hydra-opa.eb]: I corrected the syntax... update coming

**Comment by migueldiascosta** [comment]
@henkela, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**